deployment/helm/node-feature-discovery/templates/clusterrolebinding.yaml

```diff
@@ -32,3 +32,21 @@ subjects:
   name: {{ include "node-feature-discovery.topologyUpdater.serviceAccountName" . }}
   namespace: {{ include "node-feature-discovery.namespace" .  }}
 {{- end }}
+
+---
+{{- if and (eq .Values.topologyGC.enable true) (eq .Values.topologyGC.serviceAccount.create true) }}
```

Comment:
should be `rbac.create` instead of `serviceAccount.create`, right?

Reply:
good catch